src/client/app/components/window.jsx

```diff
@@ -5,10 +5,10 @@ import NumPad from "./numpad.jsx";
 class Window extends React.Component {
 
     render () {
-        const wndwStl={width:'300px', height:'300px', border:'1px solid black', borderRadius:'8px'};
+        const wndwStl = {width: "300px", height: "300px", border: "1px solid black", borderRadius: "8px", boxShadow: "0 0 5px gray"};
```

Comment:
@myxomor 
1) Change code style
2) styles in object styles.
const styles = {
wndwStl: {
width: "300px",
 height: "300px", 
border: "1px solid black", 
borderRadius: "8px", 
boxShadow: "0 0 5px gray"
},
}
3) naming "wndwStl" not readable and bad. better write container or mainContainer or calcContainer